[IMP] sales: menuselection fix

meng-odoo opened the pull request:
Fixed a menuselection error and deleted instances of second-person pov
Closes task [3116083](https://www.odoo.com/web#id=3116083&cids=3&model=project.task&view_type=form)

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/3282).

meng-odoo commented:
@StraubCreative this is ready for your review.

StraubCreative commented on content/applications/sales/crm/pipeline/multi_sales_team.rst:
```diff
@@ -2,19 +2,19 @@
+their own unique sales processes.
 
 
 Create a new Sales Team
```
Would use [sentence case](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#titles-and-headings) here, just like the H1.
```suggestion
Create a new sales team
```

StraubCreative commented on content/applications/sales/crm/pipeline/multi_sales_team.rst:
```diff
@@ -2,19 +2,19 @@
 =======================
 
-To create a new Sales Team, go to :menuselection:`CRM --> Sales --> Teams` then click
+To create a new Sales Team, go to :menuselection:`CRM --> Configuration --> Sales Teams` then click
 :guilabel:`Create`.
```
`Sales Team` doesn't need to be capitalized here, as it was already introduced as a term in the overview and it's not a proper noun.
```suggestion
To create a new sales team in Odoo, go to :menuselection:`CRM --> Configuration --> Sales Teams`
then click :guilabel:`Create`.
```

StraubCreative commented on content/applications/sales/crm/pipeline/multi_sales_team.rst:
```diff
@@ -2,19 +2,19 @@
 On the creation page, set an :guilabel:`Email Alias` to automatically generate a lead/opportunity
-for this Sales Team every time a message is sent to that unique email address. You can also choose
-whether to accept emails from :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or
+for this Sales Team every time a message is sent to that unique email address. Choose whether to
```
Lowercase
```suggestion
for this sales team every time a message is sent to that unique email address. Choose whether to
```

StraubCreative commented on content/applications/sales/crm/pipeline/multi_sales_team.rst:
```diff
@@ -2,19 +2,19 @@
+accept emails from :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or
 :guilabel:`Followers Only`.
 
 Set a :guilabel:`Domain` to assign leads/opportunities to this Sales Team based on specific
```
```suggestion
Set a :guilabel:`Domain` to assign leads/opportunities to this sales team based on specific
```

StraubCreative commented on content/applications/sales/crm/pipeline/multi_sales_team.rst:
```diff
@@ -43,8 +43,8 @@ access all of the pipelines that they need to.
 Sales Team dashboard
```
```suggestion
Sales team dashboard
```

StraubCreative reviewed (changes requested):
Hi @meng-odoo 

Good PR. Short and sweet and the RST's clean 👍 

I had only two comments:
- `sales-team-creation.png` might be just a touch too big (at ~1300px)
- `Sales Teams` is not a proper noun so I wouldn't capitalize it except when: 
    - first introducing it as a term/feature in the overview, and 
    - whenever we're calling attention to a UI element that's called `Sales Teams`

I highlighted a couple instances below, however there are much more throughout the doc. 
Can you take a look?

meng-odoo commented:
Hi @StraubCreative, I fixed the capitalization of "sales team" throughout the doc as requested. I also realized that all of the screenshots were of v14, so I updated all of them to v15. This is ready for your final review.

This PR should be forward-ported to v16 documentation, please. (We can leave v14 alone, there is no menuselection error in that doc.)

Thanks

StraubCreative commented on content/applications/sales/crm/pipeline/multi_sales_team.rst:
```diff
@@ -2,61 +2,61 @@
-maximum number of leads that can be assigned to this salesperson in a 30-day period to ensure that
-they do not overwork.
+To add team members, click :guilabel:`Add` under the :guilabel:`Members` tab when editing the sales
+team's configuration page. Select a salesperson from the dropdown menu or create new salesperson.
```
Compound modifier gets a hyphen.
```suggestion
team's configuration page. Select a salesperson from the drop-down menu or create new salesperson.
```

StraubCreative commented on content/applications/sales/crm/pipeline/multi_sales_team.rst:
```diff
@@ -2,61 +2,61 @@
 expected revenue, as well as a bar graph of new opportunities per week and an invoicing progress
 bar.
 
 .. image:: multi_sales_team/sales-team-overview.png
```
Resize + compress

StraubCreative reviewed (approved):
Hi @meng-odoo 👋 
Approving with a couple quick changes I'll push up momentarily.
Thank you!

StraubCreative commented:
@robodoo r+
